docs/tutorials/google/calibration_api.ipynb

```diff
@@ -0,0 +1,609 @@
+{
```

Comment:
<p>how can floquet compensate for chi if it can't characterize it??</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4258/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
<p>The function name is <code>make_zeta_chi_gamma_compensation_for_moments</code>,  which kind of implies that chi can be compensated. So it's a little confusing either way, but I agree your suggestion to say chi cannot be compensated is less confusing. </p>

Reply:
<p>Fixed. Also fixed some left headings in the table not being bold.</p>